compute/rn/release-information/known-issues-22-12.adoc

```diff
@@ -44,5 +44,10 @@ This issue is fixed in 22.12.582.
 +
 image::cwp-45194-rn.png[scale=15]
 
+//CWP-46005 known issue - this will be fixed in 22.12.xxx Lagrange Update 3
+* tt:[22.12.582] The count of events for WAAS agentless do not correspond to the actual number of attack requests sent.
++
+This issue is fixed in 22.12.xxx.
```

Comment:
Need to update the build number here `22.12.xxx.` at the time of release.

Reply:
good catch @divyabhushan